recipe-server/normandy/selfrepair/views.py

```diff
@@ -4,11 +4,14 @@
 from django.shortcuts import render
 from django.views.decorators.cache import cache_control
 
+from csp.decorators import csp_replace
 from rest_framework.reverse import reverse
 
 from normandy.base.decorators import short_circuit_middlewares
 
 
+# This page is primarily used by a hidden iframe in Firefox, so allow framing.
+@csp_replace(FRAME_SRC='*', FRAME_ANCESTORS='*')
```

Comment:
Is `*` needed here? It would be better to list trusted origins, such as `https://mozilla.org, https://*.mozilla.org` (just guessing, I don't know which origins are the right ones)